test/integration/command_v2/test_info_build_order.py

```diff
@@ -717,3 +717,14 @@ def test_error_different_orders(self):
         # different order
         c.run(f"graph build-order-merge --file=bo3.json --file=bo2.json", assert_error=True)
         assert "ERROR: Cannot merge build-orders of configuration!=recipe" in c.out
+
+    def test_merge_same_no_stacktrace(self):
+        tc = TestClient(light=True)
+        tc.save({"dep/conanfile.py": GenConanfile("dep", "1.0"),
+                 "pkg/conanfile.py": GenConanfile("pkg", "1.0").with_requires("dep/1.0")})
+        tc.run("export dep")
+        tc.run("export pkg")
+        tc.run("graph build-order --order=recipe --requires=pkg/1.0 --format=json", assert_error=True, redirect_stdout="order.json")
```

Comment:
`assert_error=True` because you can't call build-order and not have it fail if it will find missing binaries